README.md

```diff
@@ -172,8 +172,21 @@ it makes this module fairly useless._
 Type: `Object`  
 Default: `null`
 
-If a server instance (eg. Express or Koa) is provided, the `WebSocket` server
-will attempt to attach to the server instance instead of using a separate port.
+If a server instance is provided (e.g. what's returned when calling `.listen()` on
+an Express/Koa instance), the `WebSocket` server will attempt to attach to the
+existing server instead of using a separate port.
```

Comment:
(pasting since this was made without feedback on the proposed edit)

> e.g. what's returned when calling `.listen()` on an Express/Koa instance

Hm I find that to be an ambiguous addition. I would concede to "eg. An Express, Koa, or http.Server server instance" Beyond that, users need to have some understanding of the platform they're using if they're going to explore using an option like this.